projects/fsastorefrontlib/src/assets/translations/de/claim.de.ts

```diff
@@ -13,6 +13,14 @@ export const claim = {
     deleteStartedClaimProcess: 'Gestarteten Schadensfall löschen',
     followingClaimProcessWillBeDeleted:
       'Der folgende Schadensfall wird gelöscht.',
-    alreadySubmitted: '[DE] Your Claim is already submitted.',
+    alreadySubmitted: 'Ihr Schaden wurde bereits eingereicht.',
+    confirmation: {
+      processingNumber: 'Ihre Bearbeitungsnummer lautet',
+      submissionNotice:
+        'Ihr Schadensfall wurde erfolgreich in unserem System gespeichert. Sie finden die Bestätigungs-E-Mail im Posteingang Ihres Self-Service-Portals und in Ihrem Postfach.',
+      submissionStatus: 'To check your status online, visit Claims section.',
```

Comment:
Missed?

Reply:
Added, this is missing on develop branch as well.